packages/dx-chart-core/src/plugins/zoom-pan/computeds.ts

```diff
@@ -171,3 +171,19 @@ export const detachEvents = (node: any, handlers: EventHandlers) => {
     node.removeEventListener(el, handlers[el]);
   });
 };
+
+/** @internal */
+export const getRect = (
```

Comment:
Variable names - `prevCoords` and `nextCoords`. They are not previous and next - they are original point and current point.